desktop/cmp/input/ButtonGroupInput.ts

```diff
@@ -30,6 +31,8 @@ export interface ButtonGroupInputProps
 
     /** True to create outlined-style buttons. */
     outlined?: boolean;
+
+    ref?: ForwardedRef<any>;
```

Comment:
Trying to understand why this is needed here since it's part of the `HoistInputProps` interface.

Reply:
Well, now that ButtonGroup.ts is ForwardedRef<HTMLDivElement>, I've changed this to omit ButtonGroup's ref, and let ButtonGroupIinputProps ForwardedRef<any> apply.